docs/source/tutorial-tracking-ui.rst

```diff
@@ -0,0 +1,144 @@
+
+.. code-block:: sql
+
+  metrics.accuracy >= 0.9 AND params.model_name LIKE 'RESNET%'
+
+The target entities are:
+
+* ``metrics``: A metric logged with ``mlflow.log_metric``.
+* ``params``: A parameter logged with ``mlflow.log_param``.
+* ``tags``: A tag logged with ``mlflow.set_tag``.
+* ``attribute.run_id``: The ID of the run.
+* ``attribute.run_name``, ``run name``: The name of the run.
+* ``attribute.status``: The status of the run (``FINISHED``, ``FAILED``, ``RUNNING``, ``SCHEDULED``, ``KILLED``).
+* ``attribute.artifact_uri``: The URI of the artifact store.
+* ``attribute.user_id`` : The ID of the user who started the run.
+* ``attribute.start_time``, ``attribute.end_time`` : The start or end time of the run. Units are seconds elapsed in the Unix epoch (January 1, 1970). For example, ``start_time >= 1688169600`` filters runs created before 2023-07-01.
+
+For more, see :ref:`search-runs`.
+
+Run visibility
+~~~~~~~~~~~~~~~
+
+If, instead of defining a complete filter, you want to select among a handful of runs, you can toggle the visibility of runs in the filtered list. You do so with the column marked with an eye icon. Selecting the eye icon at the top of the column will allow you to toggle the visibility of all runs in the filtered list. Selecting the eye icon for an individual run will toggle the visibility of that run.
+
+Deleting runs
+~~~~~~~~~~~~~~
+
+You will have some runs that do not generate worthwhile results. Sometimes these will stem from code errors or other problems, but there are any number of reasons you may not want to keep a run.
+
+To delete a run, select it in the list of runs and select "Delete." You will be prompted to confirm the deletion. This puts the run in the ``Deleted`` state but *does not* remove the contents from the machine. The run is hidden by default, but can be searched for, viewed, and restored. To permanently remove the run and its associated data and artifacts, open a terminal and run:
+
+.. code-block:: bash
+
+  mlflow gc
+
+You may also delete a run using the CLI command ``mlflow run delete --run-id <run_id>`` or the Python API ``mlflow.delete_run(run_id : str)->None``.
+
+Run details page
+-----------------------
+
+Whenever you select a run in the **Table view** or **Chart view**, the run's **Detail page** opens. For a run from the **hyperparam** example, the page will look similar to this:
+
+.. image:: _static/images/ui-tutorial/run-details.png
+   :width: 100%
+
+In particular, note the **Run ID** (1), which is the primary key for the run. The run's name(for instance, ``stylish-ox-217``) is not guaranteed to be unique and cannot be used to identify a run in a program or at the command-line.
```

Comment:
Well, the chances of collision are pretty low ;) 
A good callout, though! Can we mention that setting the run name during run creation (whether through the explicit api call `mlflow.start_run()` or through the run context `with mlflow.start_run()` that the run name can be set, overriding the generated fun name that we create for users?